health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -15,85 +16,193 @@ const businessServiceMap = {};
+            const [localIsActive, setLocalIsActive] = useState(rowDataCache[serviceCode].isActive);
+            
+            const toggle = async () => {
+              const newStatus = !localIsActive;
+              const res = await updateServiceDefinition(newStatus);
+              if (res) {
+                rowDataCache[serviceCode].isActive = newStatus;
+                setLocalIsActive(newStatus);
+              }
+            };
+            
+            const switchText = localIsActive ? "Active" : "Inactive";
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)